### PR TITLE
Fixed pod errors as reported by CPANTS.

### PR DESCRIPTION
**Pull request by manwar**
Hi Team,

Please review the PR.
The change proposed to fix the possible pod errors reported by CPANTS.

https://cpants.cpanauthors.org/release/SRI/Mojolicious-8.19

Many Thanks.
Best Regards,
Mohammad S Anwar

### CODE REVIEW

File: lib/Mojo/IOLoop/Subprocess.pm [hunk 1]
```diff
@@ -124,7 +124,7 @@ can emit the following new ones.
   });
 
 Emitted in the parent process when the subprocess calls the
-L<progress|/"progress1"> method.
+L<progress|/"progress"> method.
```
Comment (haarg) [inline lib/Mojo/IOLoop/Subprocess.pm]: This doesn't really work, as there are two 'progress' sections.  The progress1 link is meant to link to the second section, documenting the method.  There isn't really a good way to "properly" link to a section like that.  It's up to however the Pod renderer decides to act.

Comment (manwar) [reply lib/Mojo/IOLoop/Subprocess.pm]: > This doesn't really work, as there are two 'progress' sections. The progress1 link is meant to link to the second section, documenting the method. There isn't really a good way to "properly" link to a section like that. It's up to however the Pod renderer decides to act.

I have removed the change.

File: lib/Mojo/Content/MultiPart.pm [hunk 2]
```diff
@@ -304,7 +304,7 @@ True, this is a L<Mojo::Content::MultiPart> object.
   my $multi
     = Mojo::Content::MultiPart->new({parts => [Mojo::Content::Single->new]});
 
-Construct a new L<Mojo::Content::MultiPart> object and subscribe to L</"read">
+Construct a new L<Mojo::Content::MultiPart> object and subscribe to L<Mojo::Content/"read">
```
Comment (kraih) [inline lib/Mojo/Content/MultiPart.pm]: Those lines look too long.

**Comment by kraih** [comment]
Thanks, i've fixed it manually since the formatting was incorrect in this PR.

**Pull request closed without merging**